app/domain/logs.rb

```diff
@@ -188,6 +188,20 @@ module AuthnAzure
       )
 
     end
+
+    module AuthnGCP
+
+      ExtractingRestrictionsFromResource = ::Util::TrackableLogMessageClass.new(
+          msg: "Extracting restrictions from resource '{0-resource-id}' with prefix '{1-prefix}'",
```

Comment:
@shulifink please review

Reply:
i think that this is confusing. we mostly refer to hosts as a role and not as a resource when it's customer facing. 
i think that this is better:
```
ExtractingResourceRestrictionsFromRole = ::Util::TrackableLogMessageClass.new(
          msg: "Extracting Resource Restrictions from role '{0-role-id}' with prefix '{1-prefix}'",
          code: "CONJ00039D"
      )